src/autoscaler/metricsforwarder/config/config_test.go

```diff
@@ -309,6 +310,55 @@ rate_limit:
 			err = conf.Validate()
 		})
 
+		When("syslog is available", func() {
+			BeforeEach(func() {
+				conf.SyslogConfig = SyslogConfig{
+					ServerAddress: "localhost",
+					Port:          514,
+					TLS: models.TLSCerts{
+						CACertFile: "../testcerts/ca.crt",
+						CertFile:   "../testcerts/client.crt",
+						KeyFile:    "../testcerts/client.crt",
+					},
+				}
+				conf.LoggregatorConfig = LoggregatorConfig{}
+			})
+
+			It("should not error", func() {
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			Context("when SyslogServer CACert is not set", func() {
```

Comment:
There are still some occurrences likes this in the test 😲 

```suggestion
			When("SyslogServer CACert is not set", func() {
```